services/queue/src/main.js

```diff
@@ -431,6 +432,7 @@ let load = loader({
         blobRegion: ctx.cfg.app.blobArtifactRegion,
         publicBlobBucket: ctx.cfg.app.publicBlobArtifactBucket,
         privateBlobBucket: ctx.cfg.app.privateBlobArtifactBucket,
+        LRUcache: new QuickLRU({maxSize: 1000000}),
```

Comment:
Let's make this a configurable value, with a small default (10?).  At a conservative 3k apiece, a million Task objects would occupy 3GB, which is more than we budget for these k8s pods.